apps/web/src/app/api/projects/[projectId]/commits/[commitUuid]/documents/[documentUuid]/documentLogs/aggregations/route.ts

```diff
@@ -0,0 +1,38 @@
+import { Workspace } from '@latitude-data/core/browser'
+import { CommitsRepository } from '@latitude-data/core/repositories'
+import { computeDocumentLogsAggregations } from '@latitude-data/core/services/documentLogs/computeDocumentLogsAggregations'
+import { authHandler } from '$/middlewares/authHandler'
+import { errorHandler } from '$/middlewares/errorHandler'
+import { NextRequest, NextResponse } from 'next/server'
+
+export const GET = errorHandler(
+  authHandler(
+    async (
+      req: NextRequest,
+      {
+        params,
+        workspace,
+      }: {
+        params: {
+          projectId: string
+          commitUuid: string
+          documentUuid: string
+        }
```

Comment:
New typed API is cooler. (mi cuña publicitaria en otras PRs)
https://github.com/latitude-dev/latitude-llm/pull/624